templates/cluster/openstack-standalone-cp/templates/k0scontrolplane.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+kind: K0sControlPlane
+metadata:
+  name: {{ include "k0scontrolplane.name" . }}
+spec:
+  k0sConfigSpec:
+    args:
+      - --enable-worker
+      - --enable-cloud-provider
+      - --kubelet-extra-args="--cloud-provider=external"
+      - --disable-components=konnectivity-server
+    k0s:
+      apiVersion: k0s.k0sproject.io/v1beta1
+      kind: ClusterConfig
+      metadata:
+        name: k0s
+      spec:
+        api:
+          extraArgs:
+            anonymous-auth: "true"
+        extensions:
+          helm:
+            repositories:
+              - name: openstack
+                url: https://kubernetes.github.io/cloud-provider-openstack/
+            charts:
+              - name: openstack-ccm
+                chartname: openstack/openstack-cloud-controller-manager
+                version: 2.31.1
+                order: 1
+                namespace: kube-system
+                values: |
+                  cloudConfig:
+                    global:
+                      auth-url: {{ .Values.openstack.authURL }}
+                      region: {{ .Values.openstack.region }}
+                      {{- if .Values.openstack.useApplicationCredentials }}
```

Comment:
Right, I think we don't need this. I thought we could remove the other fields too (since the secret would have them too), but [docs](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md#global) says that auth-url and region are required fields.